src/lib.rs

```diff
@@ -46,6 +46,9 @@
 #[macro_use]
 extern crate pretty_assertions;
 
+#[macro_use]
+extern crate log;
```

Comment:
The use of log could be another different PR.

Reply:
This one is reasonable, I can move the log migration to another PR.